src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.ts

```diff
@@ -30,7 +33,11 @@ export class ProfileCardComponent implements OnInit, OnChanges {
   constructor(private consultationsService: ConsultationsService,
               private userService: UserService,
               private cookieService: CookieService,
-              private router: Router ) { }
+              private router: Router,
+              private dialog: MatDialog
+              ){
+                import('src/app/modules/glossary/glossary.module').then(m=>m.GlossaryModule);
+              }
```

Comment:
Please move this to the module where `ProfileCardComponent` is loaded

Reply:
Done removed